delta-lake/delta-spark350db143/src/main/scala/com/nvidia/spark/rapids/delta/GpuDeltaParquetFileFormat.scala

```diff
@@ -247,8 +247,14 @@ case class GpuDeltaParquetFileFormat(
 
 object GpuDeltaParquetFileFormat {
   def tagSupportForGpuFileSourceScan(meta: SparkPlanMeta[FileSourceScanExec]): Unit = {
-    if (!meta.conf.isParquetPerFileReadEnabled) {
-      meta.willNotWorkOnGpu("Deletion vectors only supported for PERFILE reader")
+    val format = meta.wrapped.relation.fileFormat.asInstanceOf[DeltaParquetFileFormat]
```

Comment:
Let us grab protocol and metadata from fromat and call DeltectionVectorUtrils.is Supported

Reply:
why do we check for writable in the Scan acceleration? checking for readable would make sense https://github.com/delta-io/delta/blob/6118f3fd4bf4cb32abd5ebeff1e74d3d9656bd86/spark/src/main/scala/org/apache/spark/sql/delta/commands/DeletionVectorUtils.scala#L85

Reply:
That will only check if the protocol supports deletion vector and if the metadata format is parquet. Is that what we want? I thought we wanted to check the table properties?

Reply:
the table property only seems to tell us if we are allowed to add a new DV. But the existing DV data remains readable. So I guess we should just use what you had